src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -65,7 +65,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
                 subdirApp.UseEndpoints(endpoints =>
                 {
-                    endpoints.MapHub<ComponentHub>(ComponentHub.DefaultPath).AddComponent<Index>(selector: "root");
+                    endpoints.MapHub<ComponentHub>(ComponentHub.DefaultPath).AddComponent(typeof(Index), selector: "root");
```

Comment:
Huh? This looks like a move _away_ from using a generic argument

Reply:
Covered here - https://github.com/aspnet/AspNetCore/pull/8906#discussion_r270283883